src/SFA.DAS.EmployerAccounts.Web/ViewModels/CallToActionViewModel.cs

```diff
@@ -29,5 +26,7 @@ public class CallToActionViewModel
         public string HashedDraftApprenticeshipId { get; set; }
         public string HashedCohortReference { get; set; }
         public string ApprenticeName { get; set; }
+        public bool ViewApprenticeDetails => CourseStartDate.HasValue && CourseEndDate.HasValue;
```

Comment:
What does the start and end date have to do with ViewApprenticeDetails

Reply:
removed, no longer needed